.gitignore

```diff
@@ -8,4 +8,4 @@ target
 /build
 /*.iml
 /*.ipr
-/*.iws
+/*.iws
```

Comment:
Can I use the formatter at https://github.com/ndimiduk/spring-framework/blob/master/eclipse-code-formatter.xml for code formatting while contributing or let me know if you have a different location?